code/index.html

```diff
@@ -5,33 +5,82 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <link rel="stylesheet" href="style.css" />
-    <title>Technigo Project Cluedo</title>
+    <title>Cluedo</title>
+    <link rel="shortcut icon" href="./assets/splash.png" type="image/png" />
   </head>
   <body>
-    <section class="cards">
-      <article class="killer-deck" id="killerCard">
-        <p>KILLER</p>
-        <p id="killerName"></p>
-        <p id="killerAge"></p>
-        <p id="killerOccupation"></p>
-        <img id="killerImage" src="" height="100" />
```

Comment:
I did the same with you while putting the height attribute here. Some suggestion is to move it to css file instead. I am not really sure which is better or even they make any difference.

Reply:
Haha, I just made the same comment to Ylva's code!